backend/proxy-server/README.md

```diff
@@ -52,5 +52,15 @@ to have as a pre-requisite to running the test suite.
   - all field names are case sensitive for the purpose of testing
   - the fields **should** be of type `Single line text`
 
-[pre-defined pseudo-random IPs]: ../lib/fake-ips.js
+[pre-defined pseudo-random ips]: ../lib/fake-ips.js
 [script]: ../util/loopback-network.sh
```

Comment:
these lines are references to links that are specified elsewhere in the document with the same tags. they were intended to be at the end of the document like footnotes.

another approach would be to add them to the bottom of each heading section. but now we're in between both these options.